src/ring_theory/tensor_product.lean

```diff
@@ -769,6 +773,17 @@ lemma product_map_right_apply (b : B) : product_map f g (include_right b) = g b
 
 @[simp] lemma product_map_right : (product_map f g).comp include_right = g := alg_hom.ext $ by simp
 
+lemma product_map_range : (product_map f g).range = f.range ⊔ g.range :=
```

Comment:
Is this true more generally for `tensor_product.map`?

Reply:
Ah, good point. I'll take a look later today.